acceptance/tests/direct_puppet/cached_catalog_remediate_local_drift.rb

```diff
@@ -0,0 +1,112 @@
+test_name "PUP-5122: Puppet remediates local drift using code_id and content_uri"
+
+step "Add versioned-code parameters to puppetserver.conf and ensure the server is running"
+basedir = master.tmpdir("master_basedir")
```

Comment:
to make these more known, i sometimes send `File.basename(__FILE__, '.*')` as the tmpdir name.
each run is unique, but it's known what test they are for, and they can just be left hanging around for debug